stable/cluster-backup-chart/templates/odap-operator-pre-install-hook.yaml

```diff
@@ -0,0 +1,25 @@
+# Copyright Contributors to the Open Cluster Management project
+
+# pre hook used to install the OADP Operator, using the {{ .Values.oadpOperator.namespace }} namespace
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: {{ .Values.oadpOperator.name }}-subscription
+  namespace: {{ .Release.Namespace }}
+  annotations:
+    "helm.sh/hook": pre-install
+    "helm.sh/hook-weight": "-1"
+    "helm.sh/resource-policy": delete
+spec:
+  channel: {{ .Values.oadpOperator.channel }}
+  startingCSV: {{ .Values.oadpOperator.startingCSV }}
+  config:
```

Comment:
proxyConfig vars and tolerations should be added here as well

Reply:
@zkayyali812 do you have a link to a sample op I can use for the the proxyConfig and tolerations
I am not sure what exactly to add here

Reply:
I will approve this PR and we can track these additions in a separate issue